migrations/20230307200553_007.js

```diff
@@ -0,0 +1,161 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+	return knex("energizers")
```

Comment:
Is there a reason we are adding new sample data when there is already sample data created by previous migrations? Should the previous sample data be deleted first and/or updated?